DQM/DTMonitorClient/python/dtFineDelayCorr_cfi.py

```diff
@@ -36,3 +36,14 @@
 )
 
 
+from Configuration.Eras.Modifier_run2_common_cff import run2_common
+run2_common.toModify( dtFineDelayCorr, hwSources = cms.untracked.vstring('TM'))
```

Comment:
adding your change to the run2_common era is sufficient. the 25ns, run2_HI, and pA_2016 are all used only in conjunction with run2_common
@jfernan2